Kickstarter-iOS/Views/Controllers/PledgeAmountViewControllerTests.swift

```diff
@@ -137,3 +176,9 @@ final class PledgeAmountViewControllerTests: TestCase {
     }
   }
 }
+
+// MARK: - Functions
+
+private func maximumValue() -> Double {
```

Comment:
Could we have this as a static property on an enum so that we can ensure it's shared between the tests and VM? Something like:

```swift
enum PledgeAmountStepperConstants {
  static let min: Double = 0
  static let max: Double = 1_000_000_000
}
```